terraform/lib/dependabot/terraform/file_fetcher.rb

```diff
@@ -10,7 +10,7 @@ class FileFetcher < Dependabot::FileFetchers::Base
       include FileSelector
 
       def self.required_files_in?(filenames)
-        filenames.any? { |f| f.end_with?(".tf", ".tfvars") }
+        filenames.any? { |f| f.end_with?(".tf", ".hcl") }
```

Comment:
If we plan to hold off merging this, we'll want to backport this change cc @xlgmokha @Nishnha, as otherwise we'd raise an error if there is only a `.hcl` file in the repo